src/Scaffold/Console/CreateTheme.php

```diff
@@ -8,11 +8,11 @@
 class CreateTheme extends GeneratorCommand
 {
     /**
-     * The console command name.
+     * The default command name for lazy loading.
      *
-     * @var string
+     * @var string|null
      */
-    protected $name = 'create:theme';
+    protected static $defaultName = 'create:theme';
```

Comment:
What was the name property being used by previously and are we save to no longer use it?

Reply:
I believe `$name` is used in the absence of a `$signature`. It's safe to remove if we use `$defaultName` or `$signature`, as far as I can tell.